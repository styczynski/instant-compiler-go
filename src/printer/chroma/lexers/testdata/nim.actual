# -*- nim -*-
import random
proc randBool(): bool =
  randomize()
  result = rand(1).bool
for _ in 0..5:
  echo randBool()
