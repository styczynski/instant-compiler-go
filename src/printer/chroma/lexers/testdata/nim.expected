[
  {"type":"Comment","value":"# -*- nim -*-"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordNamespace","value":"import"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"random"},
  {"type":"Text","value":"\n"},
  {"type":"Keyword","value":"proc "},
  {"type":"NameFunction","value":"randBool"},
  {"type":"Punctuation","value":"():"},
  {"type":"Text","value":" "},
  {"type":"KeywordType","value":"bool"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":"\n  "},
  {"type":"Name","value":"randomize"},
  {"type":"Punctuation","value":"()"},
  {"type":"Text","value":"\n  "},
  {"type":"Name","value":"result"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Name","value":"rand"},
  {"type":"Punctuation","value":"("},
  {"type":"LiteralNumberInteger","value":"1"},
  {"type":"Punctuation","value":")."},
  {"type":"KeywordType","value":"bool"},
  {"type":"Text","value":"\n"},
  {"type":"Keyword","value":"for"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"_"},
  {"type":"Text","value":" "},
  {"type":"OperatorWord","value":"in"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberFloat","value":"0"},
  {"type":"Punctuation","value":".."},
  {"type":"LiteralNumberInteger","value":"5"},
  {"type":"Punctuation","value":":"},
  {"type":"Text","value":"\n  "},
  {"type":"Name","value":"echo"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"randBool"},
  {"type":"Punctuation","value":"()"},
  {"type":"Text","value":"\n"}
]
